backend/server/listen.py

```diff
@@ -1,312 +1,21 @@
-from server.modeltypes import DataModel, HandleFolder, HandleFile, FileWrite
-from fastapi import FastAPI, HTTPException, WebSocket, Security, Query
-from fastapi.security import APIKeyHeader
+from .routers import api, ws, api_key
+from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
-from server.session.manager import Manager
-import shutil
-import json 
-import os
 
 app = FastAPI()
 
-KEY_MAP = {"OpenAI": "OPENAI_API_KEY", "Anthropic": "ANTHROPIC_API_KEY"}
+#Include routers
+app.include_router(api_key.router, prefix="/api-key", tags=["api-key"])
```

Comment:
I think the `/api-key` route can be removed; I forgot to remove dead code from before. We handle api-key management from the websocket now